.github/actions/setup/action.yml

```diff
@@ -20,6 +20,9 @@ runs:
           packages/atomic/docs
           packages/atomic/loader
           packages/atomic/www
+          packages/atomic/src/components/search/atomic-search-interface/lang/
+          packages/atomic/src/external-builds/
+          packages/atomic/src/generated/
```

Comment:
Had some errors which I thought were because some files weren't being cached. Regardless, those files should be cached just like any other build artifact.